src/transformers/models/detr/configuration_detr.py

```diff
@@ -187,6 +188,8 @@ def __init__(
                 backbone_model_type = backbone_config.get("model_type")
                 config_class = CONFIG_MAPPING[backbone_model_type]
                 backbone_config = config_class.from_dict(backbone_config)
+            # set timm attributes to None
+            dilation, backbone, use_pretrained_backbone = None, None, None
```

Comment:
I realise this is already in the design and not introduced in this PR - but it feels a bit funny to have arguments which configure the timm checkpoint to be at the model level e.g. `dilation`, whereas for a transformers backbone they're contained within their own config.

Reply:
Yes, however note that this is a legacy way of using timm models as backbone.

In the future, one should always leverage the `AutoBackbone` class